src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Security/CertificateAuthenticationWithPki/CertificateAuthenticationWithPKI.xaml

```diff
@@ -0,0 +1,31 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.CertificateAuthenticationWithPKI.CertificateAuthenticationWithPKI"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <Grid HorizontalAlignment="Center" VerticalAlignment="Center">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="150" />
+            </Grid.ColumnDefinitions>
+            <TextBlock Text="User: "
+                       Margin="5"
+                       Grid.Row="2" Grid.Column="0" />
+            <TextBlock x:Name="LoggedInUserName" Text="Not logged in"
+                       Margin="5"
+                       Grid.Row="2" Grid.Column="1" />
+            <TextBox x:Name="PortalUrlTextbox" Text="Portal URL..."
```

Comment:
It might be good to show the format expected for the URL (assuming there is one!). I was getting a 404 with my first try ("https://portalpkids.ags.esri.com/gis/home") and had to remove the "home/" portion of the URL to get it to work.
`"https://<your_portal>/gis/"`